### PR TITLE
feat: generate certificate for business namespace

### PR DESCRIPTION
**Pull request by defydahui**

### CODE REVIEW

File: api/business/v1/types.go [hunk 1]
```diff
@@ -23,6 +23,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const CertOptionValiddays = "validdays"
```
Comment (choujimmy) [inline api/business/v1/types.go:26]: `validdays` must be `validDays`

File: api/business/v1/types.go [hunk 2]
```diff
@@ -136,6 +138,16 @@ type UsedQuantity struct {
...
 type ClusterUsed map[string]UsedQuantity
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NamespaceCertOptions is query options of getting namespace with a x509 certificate.
+type NamespaceCertOptions struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// Pay attention to const CertOptionValiddays!
+	ValidDays string `json:"validdays,omitempty" protobuf:"bytes,1,opt,name=validdays"`
```
Comment (choujimmy) [inline api/business/v1/types.go:148]: json tag name must be `validDays`

File: api/business/v1/types.go (hunk 1 above)
Comment (defydahui) [reply api/business/v1/types.go]: Done. Plz review again.

File: api/business/v1/types.go (hunk 2 above)
Comment (defydahui) [reply api/business/v1/types.go]: Done. Plz review again.

**Pull request merged**